src/weekly-calendar/types.ts

```diff
@@ -0,0 +1,15 @@
+import { CommonCalendarProps } from "../shared/internal-calendar/types";
+import { TimeSlot } from "../time-slot-bar";
+
+export interface WeeklyCalendarProps extends CommonCalendarProps {
+    showNavigationHeader?: boolean; // default true: show or hide month/year and arrow dropdowns
```

Comment:
instead of a `showNavigationHeader` prop, the TimeSlotWeekView can be exported as a standalone component as well?